public/img/lines-after.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="46" height="58" viewBox="0 0 46 58" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If we delete the stroke & fill color on all of these SVGs we can then change their color in the css!